tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -28,21 +32,48 @@ partial class Build
     [Parameter("Indicates if the Dynamic Instrumentation product should be disabled.")]
     readonly bool DisableDynamicInstrumentationProduct;
 
+    [Parameter("Indicates whether the line probe scenario should run.")]
+    readonly bool LineProbes;
```

Comment:
I take it you still want this as a parameter? 🙂

Reply:
Fix in 9f23fdedaa2a7645d2ae2d2edbaf79936c9e57ef